client/task-list/tasks.js

```diff
@@ -226,8 +227,15 @@ export function getAllTasks( {
 			title: __( 'Set up shipping costs', 'woocommerce-admin' ),
 			container: <Shipping />,
 			onClick: () => {
-				onTaskSelect( 'shipping' );
-				updateQueryString( { task: 'shipping' } );
+				if ( shippingZonesCount > 0 ) {
+					window.location = getLinkTypeAndHref( {
```

Comment:
One very minor issue with this is that we can't ctrl/cmd + click to open this up in a new tab.  This would require some updates to the task list to allow links instead of click handlers, so I'm not sure it's worth the time at the moment to add this for a single task.

Reply:
Yeah that would indeed be nice, there is some task list refactoring coming down the pipe, this one being the first: https://github.com/woocommerce/woocommerce-admin/pull/6787
So hopefully we could add it with that refactor.